### PR TITLE
Add words from misspell's dictionary (A-1)

### PR DESCRIPTION
**Pull request by skangas**
This adds corrections from misspell's wordlist:
https://github.com/client9/misspell/blob/master/words.go

misspell is distributed under the MIT license:
https://github.com/client9/misspell/blob/master/LICENSE

These are the first 500 words from that dictionary, merged into ours and with corrections and additions by me.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -810,12 +1182,18 @@ acumulator->accumulator
 acuqire->acquire
 acuracy->accuracy
 acurate->accurate
+acusation->accusation, causation,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:1185]: Please add: 
`acusations->accusations, causations,`

Comment (skangas) [reply codespell_lib/data/dictionary.txt:1185]: Now done, thanks!

**Review by luzpaz — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
 aautomatically->automatically
 abailable->available
 abanden->abandon
+abandenment->abandonment
+abandining->abandoning
+abandomnent->abandonment
+abandond->abandoned
 abandonded->abandoned
+abandonding->abandoning
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:41]: Perhaps add `abandonds->abandons` too?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
...
 abandone->abandon
+abandonig->abandoning
+abandonne->abandonment, abandon,
 abandonned->abandoned
+abandonnent->abandonment
 abandonning->abandoning
+abanonded->abandoned
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:48]: Perhaps also:
```
abanond->abandon
abanonding->abandoning
abanonds->abandons
```
I have not found `abanonding` in GitHub, only in the internet at large with Google.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
...
 abberration->aberration
+abberrations->aberrations
+abberviation->abbreviation
+abberviations->abbreviations
 abbort->abort, abbot,
 abborted->aborted
 abborting->aborting
 abborts->aborts, abbots,
 abbout->about, abbot,
+abbreivation->abbreviation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:64]: Plural `abbreivations->abbreviations` too

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
...
+abbreviatins->abbreviations
+abbreviato->abbreviation
 abbreviaton->abbreviation
 abbreviatons->abbreviations
+abbrievation->abbreviation
+abbrievations->abbreviation
 abbriviate->abbreviate
 abbriviation->abbreviation
 abbriviations->abbreviations
+abdominable->abdominal
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:81]: Perhaps `abdominable->abdominal, abominable,` although the second is certainly less probable?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -77,20 +105,45 @@ abnornally->abnormally
...
+aboriginial->aboriginal
+aborigional->aboriginal
+aborignial->aboriginal
+aborigonal->aboriginal
+aboroginal->aboriginal
 aborte->aborted, abort, aborts,
 abortificant->abortifacient
 aboslute->absolute
 aboslutely->absolutely
+aboslutes->absolutes
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:137]: `absolute` is an adjective, why the plural here?

Of course, adjectives can be used as nouns in spoken English, but that's not really correct English, is it? In any case, the [nominalized](https://www.merriam-webster.com/dictionary/nominalize) or [substantivized](https://www.merriam-webster.com/dictionary/substantivize) adjective usually remains singular. There have already been debates on the subject here. See:
* [Adjectives that Look Like Nouns](https://www.merriam-webster.com/words-at-play/adjectives-that-look-like-nouns)
* [Adjectives Function as Nouns](http://myenglishgrammar.com/lesson-3-adjectives/4-adjectives-function-as-nouns.html)
* [Are adjectives the new nouns?](https://www.macmillandictionaryblog.com/are-adjectives-the-new-nouns)
* [Advanced English Lesson: Using ADJECTIVES as NOUNS](https://www.engvid.com/adjectives-as-nouns/)
* [How to use Adjectives as Nouns](https://worldenglishblog.com/use-adjectives-as-nouns/)

But then I am not a native speaker.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -114,9 +167,17 @@ abreviates->abbreviates
 abreviating->abbreviating
 abreviation->abbreviation
 abreviations->abbreviations
+abriter->arbiter
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:170]: Add plural `abriters->arbiters`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -33,34 +33,62 @@ aautomatic->automatic
...
 abbrevations->abbreviations
+abbreveation->abbreviation
+abbreveations->abbreviations
+abbreviatin->abbreviation, abbreviating,
+abbreviatins->abbreviations
+abbreviato->abbreviation
 abbreviaton->abbreviation
 abbreviatons->abbreviations
+abbrievation->abbreviation
+abbrievations->abbreviation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:77]: Add verb?
```
abbrievate->abbreviate
abbrievated->abbreviated
abbrievates->abbreviates
abbrievating->abbreviating
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -114,9 +167,17 @@ abreviates->abbreviates
 abreviating->abbreviating
 abreviation->abbreviation
 abreviations->abbreviations
+abriter->arbiter
 abritrarily->arbitrarily
 abritrary->arbitrary
+abritration->arbitration
 abriviate->abbreviate
+abrreviation->abbreviation
+abrreviations->abbreviations
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:176]: Add verb?
```
abrreviate->abbreviate
abrreviated->abbreviated
abrreviates->abbreviates
abrreviating->abbreviating
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -125,6 +186,11 @@ abscound->abscond
 abselutely->absolutely
 abselutly->absolutely
 absense->absence
+absestos->asbestos
+absintence->abstinence
+absitnence->abstinence
+absloute->absolute
+absloutes->absolutes
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:193]: Again, why the plural `absolutes`?

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -151,29 +217,38 @@ absoluetly->absolutely
...
 absolurely->absolutely
 absolut->absolute
 absolutelly->absolutely
+absolutelys->absolutes
 absolutey->absolutely, absolute,
 absoluth->absolute
 absoluthe->absolute
 absoluthely->absolutely
 absoluthly->absolutely
+absolutisme->absolutes, absolutism,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:233]: Perhaps plain
```
absolutisme->absolutism
absolutiste->absolutist
```
In any case, put `absolutes` second.

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -295,11 +418,18 @@ acccuracy->accuracy
...
 accecpt->accept
 accecpted->accepted
+accedent->accident
+accedental->accidental
+accedentally->accidentally
+accedents->accidents
 accees->access
 acceess->access
 accelarate->accelerate
+accelarated->accelerated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:431]: `accelarates->accelerates`

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -310,16 +440,33 @@ acceleartes->accelerates
 acceleartion->acceleration
 acceleartor->accelerator
 acceleated->accelerated
+accelerade->accelerated
+acceleraptor->accelerator
+accelerar->accelerator
+accelerare->accelerate
+accelerater->accelerator
+acceleratie->accelerate
+acceleratio->accelerator
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:449]: `acceleratio->accelerator, acceleration,`

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -310,16 +440,33 @@ acceleartes->accelerates
...
+accelerato->acceleration
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:450]: accelerato->accelerator, acceleration,

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -439,14 +615,20 @@ accomidate->accommodate
 accommadate->accommodate
 accommadates->accommodates
 accommadating->accommodating
+accommadation->accommodation
 accommdated->accommodated
+accommidate->accommodate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:620]: All verbal forms?
```
accommidated->accommodated
accommidates->accommodates
accommidating->accommodating
```

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -439,14 +615,20 @@ accomidate->accommodate
 accommadate->accommodate
 accommadates->accommodates
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:616]: Add `accommadated->accommodated`?

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -439,14 +615,20 @@ accomidate->accommodate
...
+accommidate->accommodate
+accommidation->accommodation
 accomodata->accommodate
 accomodate->accommodate
 accomodated->accommodated
 accomodates->accommodates
 accomodating->accommodating
 accomodation->accommodation
 accomodations->accommodations
+accomondate->accommodate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:629]: All verbal forms?
```
accomondated->accommodated
accomondates->accommodates
```

File: codespell_lib/data/dictionary.txt [hunk 18]
```diff
@@ -531,12 +745,25 @@ accually->actually
 accuarcy->accuracy
 accuarte->accurate
 accuartely->accurately
+accuastion->accusation
+acculumate->accumulate
+acculumated->accumulated
+acculumates->accumulates
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:751]: `acculumating->accumulating`

File: codespell_lib/data/dictionary.txt [hunk 19]
```diff
@@ -531,12 +745,25 @@ accually->actually
...
+acculumation->accumulation
 accumalate->accumulate
+accumalated->accumulated
 accumalates->accumulates
+accumalation->accumulation
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:756]: `accumalating->accumulating`

File: codespell_lib/data/dictionary.txt [hunk 20]
```diff
@@ -531,12 +745,25 @@ accually->actually
...
 accumalator->accumulator
 accumalte->accumulate
 accumalted->accumulated
+accumelate->accumulate
+accumelated->accumulated
+accumelates->accumulates
+accumelating->accumulating
+accumilate->accumulate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:764]:
```
accumilates->accumulates
accumilating->accumulating
```

File: codespell_lib/data/dictionary.txt [hunk 21]
```diff
@@ -606,13 +847,30 @@ acheives->achieves
...
+achilleos->Achilles
+achilleous->Achilles
+achilleus->Achilles
 achitecture->architecture
 achitectures->architectures
 achivable->achievable
 achive->achieve, archive,
+achiveable->achievable
 achived->achieved, archived,
+achiveing->achieving
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:873]: `achives->achieves, archives,`

File: codespell_lib/data/dictionary.txt [hunk 22]
```diff
@@ -647,26 +910,60 @@ acknoledges->acknowledges
...
+acknolwedged->acknowledged
+acknolwedgement->acknowledgement
+acknolwedges->acknowledges
+acknolwedging->acknowledging
+acknoweldge->acknowledge
+acknoweldged->acknowledged
+acknoweldgement->acknowledgement
+acknoweldges->acknowledges
+acknowiedged->acknowledged
+acknowladges->acknowledges
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:923]:
```
acknowladge->acknowledge
acknowladged->acknowledged
acknowladging->acknowledging
```

File: codespell_lib/data/dictionary.txt [hunk 23]
```diff
@@ -647,26 +910,60 @@ acknoledges->acknowledges
...
+acknolwedge->acknowledge
+acknolwedged->acknowledged
+acknolwedgement->acknowledgement
+acknolwedges->acknowledges
+acknolwedging->acknowledging
+acknoweldge->acknowledge
+acknoweldged->acknowledged
+acknoweldgement->acknowledgement
+acknoweldges->acknowledges
+acknowiedged->acknowledged
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:922]:
```
acknowiedge->acknowledge
acknowiedges->acknowledges
acknowiedging->acknowledging
```

File: codespell_lib/data/dictionary.txt [hunk 24]
```diff
@@ -647,26 +910,60 @@ acknoledges->acknowledges
...
 acknoledgments->acknowledgments
+acknolwedge->acknowledge
+acknolwedged->acknowledged
+acknolwedgement->acknowledgement
+acknolwedges->acknowledges
+acknolwedging->acknowledging
+acknoweldge->acknowledge
+acknoweldged->acknowledged
+acknoweldgement->acknowledgement
+acknoweldges->acknowledges
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:921]: `acknoweldging->acknowledging`

File: codespell_lib/data/dictionary.txt [hunk 25]
```diff
@@ -647,26 +910,60 @@ acknoledges->acknowledges
...
+acknowiedged->acknowledged
+acknowladges->acknowledges
 acknowldeged->acknowledged
 acknowldegement->acknowledgement
 acknowldegements->acknowledgements
+acknowldge->acknowledge
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:927]:
```
acknowldged->acknowledged
acknowldges->acknowledges
acknowldging->acknowledging
```

File: codespell_lib/data/dictionary.txt [hunk 26]
```diff
@@ -687,45 +984,89 @@ acontiguous->a contiguous
...
+acquiantances->acquaintances
+acquianted->acquainted
 acquiantence->acquaintance
 acquiantences->acquaintances
 acquiesence->acquiescence
 acquiess->acquiesce
 acquiessed->acquiesced
 acquiesses->acquiesces
 acquiessing->acquiescing
+acquifer->acquire
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1033]: `acquifer->aquifer, acquire,`?

File: codespell_lib/data/dictionary.txt [hunk 27]
```diff
@@ -687,45 +984,89 @@ acontiguous->a contiguous
...
+acronysm->acronyms
 acroos->across
 acrosss->across
 acrost->across
+acroynm->acronym
+acroynms->acronyms
 acrue->accrue
 acrued->accrued
 acryllic->acrylic
+acrynoms->acronyms
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1064]: Singular `acrynom->acronym`?

File: codespell_lib/data/dictionary.txt [hunk 28]
```diff
@@ -746,24 +1088,44 @@ actiavtions->activations
...
 acticate->activate
+acticating->activating
+actication->activation
 actice->active
 actine->active
 actiual->actual
 activ->active
+activacion->activation
 activaed->activated
+activaste->activate, activates,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1099]:
```
activasted->activated
activastes->activates
activasting->activating
```

File: codespell_lib/data/dictionary.txt [hunk 29]
```diff
@@ -746,24 +1088,44 @@ actiavtions->activations
...
+activateing->activating
+activaters->activates
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1101]: `activaters->activates, activators, actuators`

File: codespell_lib/data/dictionary.txt [hunk 30]
```diff
@@ -746,24 +1088,44 @@ actiavtions->activations
...
+activatin->activating, activation,
 activationg->activating
 actived->activated
+activelly->actively
 activeta->activate
 activete->activate
 activeted->activated
 activetes->activates
 activiate->activate
+activiates->activist
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1111]: `activiates->activates, activist,`

**Review by DimitriPapadopoulos — COMMENTED** [review]
A few suggestions, mainly:
* singular-plural
* all verbal forms

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:77]: And change:
```
abbrievations->abbreviation
```
to:
```
abbrievations->abbreviations
```

File: codespell_lib/data/dictionary.txt [hunk 31]
```diff
@@ -606,13 +814,31 @@ acheives->achieves
...
+achievemnt->achievement
+achievemnts->achievements
+achievemts->achieves
+achievents->achieves
 achievment->achievement
 achievments->achievements
+achillees->achilles
+achilleos->achilles
+achilleous->achilles
+achilleus->achilles
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt]: `Achilles`

File: codespell_lib/data/dictionary.txt [hunk 32]
```diff
@@ -1134,15 +1182,16 @@ acumulator->accumulator
...
 acused->accused
-acusing->causing
+acusing->causing, accusing,
 acustom->accustom
 acustommed->accustomed
 acutal->actual
 acutality->actuality
 acutally->actually
-acutaly->actualy
+acutaly->actually
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1194]: `acutely` too.

**Comment by DimitriPapadopoulos** [comment]
@skangas I'm afraid I messed up trying to add my own fixes.

**Comment by DimitriPapadopoulos** [comment]
I can fix that for you, or let you handle it, at your convenience. I am terribly sorry about that.

**Pull request merged**